overlord/snapstate/handlers.go

```diff
@@ -1202,23 +1176,13 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) (err error) {
 		return err
 	}
 
-	// get the services which LinkSnap should disable when generating wrappers,
```

Comment:
To be clear, when we run `link-snap` we do not save LastActiveDisabledServices anymore because that is handled in start-snap-services instead? 

Conversely, we don't restore LastActiveDisabledServices on undo of `link-snap` anymore because that is handled in `stop-snap-services` which is run when `start-snap-services` needs to be undone.